src/components/atoms/IconInput/IconInput.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ChangeEvent } from "react";
```

Comment:
@mike-lvov @mcflyok etc: Do you know if this counts as an `atom`? Or should it be a `molecule` or something else? (i'm still not perfectly sure what should be where)

- `src/components/atoms/IconInput/IconInput.tsx`

Reply:
I think basic components: Button/Input/Text should be considered `atoms`, even if they have some additional logic inside

Reply:
_nods_ cool, works for me.